track/http_client_test.go

```diff
@@ -33,13 +33,13 @@ func TestHTTPClientRequestTracking(t *testing.T) {
 
 	gc.Convey("Client should properly track all bytes transferred for an HTTP request", t, func() {
 		client := NewDefaultHTTPClient()
-		defer client.CloseIdleConnections()
+		defer client.Client.CloseIdleConnections()
```

Comment:
The compiler was complaining about these calls: `ambiguous selector client.CloseIdleConnectionscompiler`